app/presenters/hyrax/admin_set_presenter.rb

```diff
@@ -3,5 +3,22 @@ class AdminSetPresenter < CollectionPresenter
     def total_items
       ActiveFedora::SolrService.count("{!field f=isPartOf_ssim}#{id}")
     end
+
+    # AdminSet cannot be deleted if default set or non-empty
+    def disable_delete?
+      total_items > 0 || default_set?
+    end
+
+    # Message to display if deletion is disabled
+    def disabled_message
+      return I18n.t('hyrax.admin.admin_sets.delete.error_default_set') if default_set?
```

Comment:
This may be nitpicky: I wonder if given the order of `return`s here we ought to reorganize the `disable_delete?` check above to check for `default_set?` to the left of the `total_items > 0` check? If I try to delete a default admin set with 1 item, there is a chance that the `disable_delete?` will trigger because of `total_items` and show me the message about the default set. Like I said, nitpicky and possibly not worth doing.